apps/happy-blocks/src/pricing-plans/test/edit.test.jsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { render } from '@testing-library/react';
+import { useBlockProps } from '@wordpress/block-editor';
+import { Edit } from '../edit';
+
+jest.mock( '@wordpress/block-editor', () => ( {
+	...jest.requireActual( '@wordpress/block-editor' ),
+	useBlockProps: jest.fn(),
+} ) );
+
+describe( 'UpgradeBlock/Edit', () => {
+	beforeEach( () => {
+		useBlockProps.mockClear();
+	} );
+
+	test( 'should render', () => {
+		const wrapper = render( <Edit /> );
```

Comment:
One little nit here that will help keep the snapshot smaller: instead of keeping the entire object in the snapshot, we could just use the markup:

```suggestion
		const { container } = render( <Edit /> );
```

That will rid us of all the underlying methods that exist in the object returned by `render()`.

It will also help follow the [`testing-library/render-result-naming-convention`](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/render-result-naming-convention.md) ESLint rule, which isn't enabled right now, but ideally should be when we use testing library.

Reply:
Fix in https://github.com/Automattic/wp-calypso/pull/69824